static/src/javascripts/projects/common/modules/experiments/tests/participation-low-fric-emotion.js

```diff
@@ -0,0 +1,129 @@
+define([
+    'common/utils/config',
+    'common/utils/mediator',
+    'common/utils/detect',
+    'Promise',
+    'common/modules/experiments/low-friction-participation',
+    'inlineSvg!svgs/icon/emotion-happy',
```

Comment:
Yes, very good! and if you ever need to decorate the `SPAN` with classes, you can use `common/views/svg` (without s) 👍